### ISSUE TITLE
Some Words Appear In solutions.lua But Not words.lua - Impossible to Enter

### ISSUE THREAD

**Issue by julieshah24** [HTV04/wiirdle#1]
The word "VSNET" (solution for 11/10) appears in solutions.lua but not words.lua. We are convinced that this is the only possible solution to this puzzle and have included a screenshot of our guess attempts. We are not allowed to enter "VSNET" which we believe to be because it is not in the words list. 

![PXL_20221111_050527098](https://user-images.githubusercontent.com/51678967/201268035-9e8e0838-5ac4-4c0e-a4bf-da864ae20686.jpg)

There are other words that are in the solution list that aren't in the words list. "ENDIF" and "DICKE" are the other ones we've found.

**Comment by loportol** [comment]
bump

**Comment by htv04** [comment]
Yeah, I really need to revise the word list at some point, will keep this open and pinned until then
